site/sfguides/src/getting_started_with_traces/getting_started_with_traces.md

```diff
@@ -0,0 +1,443 @@
+
+Click on Trace for the Notebook. Select the Span Type Filters at the top to include UDF and Procedures. Excluding Streamlit will remove the notebook calls from the view. Look for the `get_weather_for_cities_sp` span and the `__main__:get_weather_for_all_cities` spans which are the more time consuming parts of the procedure. You can see fetching the data from the API and saving is only a few seconds, but the entire runtime of the procedure is over a minute.
+
+If you expand the `get_weather_fn` entry under `__main__:get_weather_for_all_cities`, you can see the Debug Logs by clicking on Logs on the trace to see all the http request information.
+
+This tracing information shows the entire execution timeline with information on every call.
+
+<img src="assets/trace_details.png" width="889" height="513">
+
+<!-- ------------------------ -->
+## Query the Metrics, Logs, and Trace Data
+Duration: 3
+
+Memory and CPU metrics are also available because the session set the METRIC_LEVEL to all.
+
+To get to this data, it is in the Events table.
+
+Find your events table by running this sql:
+
+```sql
+SHOW PARAMETERS LIKE 'event_table' IN ACCOUNT;
+
+```
+
+Query the memory and cpu metrics with this query, replacing `YOUR_EVENT_TABLE` with your event table found in the previous query:
+
+```sql
+SELECT *
+FROM YOUR_EVENT_TABLE
+WHERE
+    RECORD_TYPE = 'METRIC' 
+    and RESOURCE_ATTRIBUTES['db.user'] = '{{user_name}}' 
+    and RESOURCE_ATTRIBUTES['snow.database.name'] = '{{database_name}}' 
+    and RESOURCE_ATTRIBUTES['snow.schema.name'] = '{{schema_name}}'
+ORDER BY TIMESTAMP DESC;
+
+```
+
+Query the traces with this query, replacing `YOUR_EVENT_TABLE` with your event table found in the previous query:
+
+```sql
+SELECT *
+FROM YOUR_EVENT_TABLE
+WHERE
+    RECORD_TYPE = 'SPAN' 
+    and RESOURCE_ATTRIBUTES['db.user'] = '{{user_name}}' 
+    and RESOURCE_ATTRIBUTES['snow.database.name'] = '{{database_name}}' 
+    and RESOURCE_ATTRIBUTES['snow.schema.name'] = '{{schema_name}}'
+ORDER BY TIMESTAMP DESC;
+
+```
+
+Query the logs with this query, replacing `YOUR_EVENT_TABLE` with your event table found in the previous query:
+
+```sql
+SELECT *
+FROM YOUR_EVENT_TABLE
+WHERE
+    RECORD_TYPE = 'LOG' 
+    and RESOURCE_ATTRIBUTES['db.user'] = '{{user_name}}' 
+    and RESOURCE_ATTRIBUTES['snow.database.name'] = '{{database_name}}' 
+    and RESOURCE_ATTRIBUTES['snow.schema.name'] = '{{schema_name}}'
+ORDER BY TIMESTAMP DESC;
+
+```
+
+<!-- ------------------------ -->
+## Disabling Traces and Metrics
+Duration: 2
+
+Changing the level of tracing, metrics, and logs can be done similarly to how we enabled it, with SQL.
+
+Add a new SQL Cell and paste in the following contents and run.
+
+```sql
+ALTER DATABASE "{{database_name}}" SET LOG_LEVEL = WARN;
+ALTER SCHEMA "{{schema_name}}" SET LOG_LEVEL = WARN;
```

Comment:
nit: These should be `OFF` as well. 
https://docs.snowflake.com/en/sql-reference/parameters#log-level

Reply:
why, i'd run any production workload in warn or error without concern.